plugins/modules/utm_proxy_frontend.py

```diff
@@ -207,7 +207,7 @@
       description: State of html rewrite.
       type: bool
     htmlrewrite_cookies:
-      description: Whether the html rewrite cookie will be set.
+      description: Whether the html rewrite cookie is set.
```

Comment:
Another rewrite for andebox:
```suggestion
      description: Whether the HTML rewrite cookie is set.
```